src/main/java/net/rptools/maptool/client/ui/theme/ThemeSupport.java

```diff
@@ -643,10 +643,16 @@ public static ImageIcon getExampleImageIcon(String themeName, Dimension dimensio
         || themeDetails.imagePath.isEmpty()) {
       return new ImageIcon();
     } else {
+      var imageLocation = IMAGE_PATH + themeDetails.imagePath;
+      log.info("Retrieving resource for theme name={} from location={}", themeName, imageLocation);
+      var imageURL = ThemeSupport.class.getResource(imageLocation);
+      if (imageURL == null) {
+        log.warn(
+            "Failed to retrieve resource for theme name={} from url={}, using empty ImageIcon");
+        return new ImageIcon();
+      }
       var imageIcon =
-          new ImageIcon(
-              ThemeSupport.class.getResource(IMAGE_PATH + themeDetails.imagePath),
-              themeDetails.name);
+          new ImageIcon(ThemeSupport.class.getResource(imageLocation), themeDetails.name);
```

Comment:
We can use `imageURL` here as well for a bit more clarity.

Reply:
Thanks for pointing this out, I actually wanted to do that!